www/redland-bindings/Portfile

```diff
@@ -35,33 +32,40 @@ pre-configure {
     }
 }
 
-variant perl5 {
+default_variants +python37
```

Comment:
I think one should only use this variant if no other is specified by the user, currently it will always add {{{python37}}}. So the code should/could become like this:
```
if {![variant_isset perl5] && ![variant_isset php5] && ![variant_isset python27] && ![variant_isset python36] &&
    ![variant_isset perl5] && ![variant_isset ruby] && ![variant_isset tcl] && ![variant_isset lua]} {
        default_variants +python37
}
```